source/mir/random/tinflex/internal/transformations.d

```diff
@@ -0,0 +1,147 @@
+module mir.random.tinflex.internal.transformations;
```

Comment:
Please rewrite this module using `switch`, `pow`, and `copysign`. It contains computation errors!